dandiapi/api/tests/test_tasks.py

```diff
@@ -120,13 +120,6 @@ def test_write_assets_yaml_already_exists(storage: Storage, version: Version, as
 def test_validate_asset_metadata(version: Version, asset: Asset):
     version.assets.add(asset)
 
-    # This is the minimum asset metadata for schema version 0.3.0
-    asset.metadata.metadata = {
-        'schemaVersion': '0.3.0',
-        'encodingFormat': 'application/x-nwb',
-    }
-    asset.metadata.save()
-
```

Comment:
There are several places in this file for this old format for metadata is still used. It'd probably be good (for consistency) to also change them to only add/modify the things that they need to.

Reply:
I don't want to dilute this PR more than it already is, so I will file an issue